Source/controls/plrctrls.cpp

```diff
@@ -60,7 +60,7 @@ bool InGameMenu()
 	    || qtextflag
 	    || gmenu_is_active()
 	    || PauseMode == 2
-	    || MyPlayer->_pInvincible;
+	    || (MyPlayer->_pInvincible && MyPlayer->_pHitPoints == 0);

```

Comment:
Should this be check for shifted hp?
or even <= 0 ?

Reply:
Of the places that check the `_pInvincible` flag, there is one location in monster.cpp that uses `player._pHitPoints >> 6 <= 0`. Nine other locations use `player._pHitPoints == 0`. Since it's simpler and doesn't appear to cause any problems, I went with this.